### PR TITLE
[ID] Update `Reporting bad behaviour`

### PR DESCRIPTION
**Pull request by mousesalt**
ah yes, community must articles must be updated as well

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Reporting_bad_behaviour/id.md [hunk 1]
```diff
@@ -13,10 +8,11 @@ Laporan yang dikirim secara tidak sengaja akan diabaikan. Meskipun demikian, par
 
 | Jenis pelanggaran | Pihak yang dapat dihubungi |
 | :-- | :-- |
-| Pesan pribadi yang tidak pantas | **Tidak ada**: harap [abaikan pengguna tersebut](/wiki/Client/Interface/Chat_console#daftar-perintah). Untuk dapat mengabaikan seseorang secara permanen, gunakan fitur daftar pengabaian (*ignore list*) yang ada pada menu [Options](/wiki/Client/Options). |
-| Pelanggaran yang terkait dengan modding dan/atau mapping | NAT atau para anggota [GMT](/wiki/People/The_Team/Global_Moderation_Team) yang memoderasi komunitas mapping/modding, melalui situs web [NAT/BN Management](https://bn.mappersguild.com/reports) |
-| Perilaku yang tidak pantas dari anggota [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) atau [GMT](/wiki/People/The_Team/Global_Moderation_Team) | [Account support team](/wiki/People/The_Team/Account_support_team) melalui alamat email [support@ppy.sh](mailto:support@ppy.sh) |
-| Pelecehan seksual serta segala bentuk kelakuan buruk yang sudah tidak dapat ditoleransi lagi | Harap rujuk artikel [Melaporkan tindak pelecehan](/wiki/Reporting_bad_behaviour/Abuse) |
+| Scamming/Phishing di dalam pesan pribadi | Kirim laporan biasa |
```
Comment (Nivalyx) [inline wiki/Reporting_bad_behaviour/id.md:11]:
```suggestion
| Upaya *scamming*/*phishing* di dalam pesan pribadi | Kirimkan laporan seperti biasa |
```

File: wiki/Reporting_bad_behaviour/id.md [hunk 2]
```diff
@@ -13,10 +8,11 @@ Laporan yang dikirim secara tidak sengaja akan diabaikan. Meskipun demikian, par
...
+| Pesan pribadi yang menyinggung/tidak pantas | **Tidak ada**: harap [abaikan pengguna tersebut](/wiki/Client/Interface/Chat_console#daftar-perintah). Untuk dapat mengabaikan seseorang secara permanen, gunakan fitur daftar pengabaian (*ignore list*) yang ada pada menu [Options](/wiki/Client/Options) atau blokir pengguna tersebut melalui tombol di website pada profil mereka |
+| Perilaku yang tidak pantas dari anggota [BN](/wiki/People/The_Team/Beatmap_Nominators) | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team), melalui website [NAT/BN Management](https://bn.mappersguild.com/reports) |
+| Perilaku yang tidak pantas dari anggota [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) atau [GMT](/wiki/People/The_Team/Global_Moderation_Team) | [Account support team](/wiki/People/The_Team/Account_support_team) melalui [support@ppy.sh](mailto:support@ppy.sh) |
+| Pelecehan seksual atau perilaku ekstrem | Mengacu pada [Reporting Abuse](/wiki/Reporting_bad_behaviour/Abuse) |
```
Comment (Nivalyx) [inline wiki/Reporting_bad_behaviour/id.md:15]:
```suggestion
| Pelecehan seksual atau perilaku menyimpang lainnya yang berlebihan | Harap rujuk artikel [[Melaporkan tindak pelecehan](https://osu.ppy.sh/wiki/id/Reporting_bad_behaviour/Abuse)](/wiki/Reporting_bad_behaviour/Abuse) |
```

File: wiki/Reporting_bad_behaviour/id.md [hunk 3]
```diff
@@ -13,10 +8,11 @@ Laporan yang dikirim secara tidak sengaja akan diabaikan. Meskipun demikian, par
...
-| Pelanggaran yang terkait dengan modding dan/atau mapping | NAT atau para anggota [GMT](/wiki/People/The_Team/Global_Moderation_Team) yang memoderasi komunitas mapping/modding, melalui situs web [NAT/BN Management](https://bn.mappersguild.com/reports) |
-| Perilaku yang tidak pantas dari anggota [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) atau [GMT](/wiki/People/The_Team/Global_Moderation_Team) | [Account support team](/wiki/People/The_Team/Account_support_team) melalui alamat email [support@ppy.sh](mailto:support@ppy.sh) |
-| Pelecehan seksual serta segala bentuk kelakuan buruk yang sudah tidak dapat ditoleransi lagi | Harap rujuk artikel [Melaporkan tindak pelecehan](/wiki/Reporting_bad_behaviour/Abuse) |
+| Scamming/Phishing di dalam pesan pribadi | Kirim laporan biasa |
+| Pesan pribadi yang menyinggung/tidak pantas | **Tidak ada**: harap [abaikan pengguna tersebut](/wiki/Client/Interface/Chat_console#daftar-perintah). Untuk dapat mengabaikan seseorang secara permanen, gunakan fitur daftar pengabaian (*ignore list*) yang ada pada menu [Options](/wiki/Client/Options) atau blokir pengguna tersebut melalui tombol di website pada profil mereka |
```
Comment (Nivalyx) [inline wiki/Reporting_bad_behaviour/id.md:12]:
```suggestion
| Pesan pribadi yang menyinggung/tidak pantas | **Tidak ada**: harap [abaikan pengguna tersebut](/wiki/Client/Interface/Chat_console#daftar-perintah). Untuk dapat mengabaikan seseorang secara permanen, gunakan fitur daftar pengabaian (*ignore list*) yang ada pada menu [Options](/wiki/Client/Options) atau blokir pengguna tersebut melalui tombol yang ada pada laman profil mereka. |
```

File: wiki/Reporting_bad_behaviour/id.md [hunk 4]
```diff
@@ -13,10 +8,11 @@ Laporan yang dikirim secara tidak sengaja akan diabaikan. Meskipun demikian, par
...
+| Perilaku yang tidak pantas dari anggota [BN](/wiki/People/The_Team/Beatmap_Nominators) | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team), melalui website [NAT/BN Management](https://bn.mappersguild.com/reports) |
```
Comment (Nivalyx) [inline wiki/Reporting_bad_behaviour/id.md:13]:
```suggestion
| Perilaku yang tidak pantas dari anggota [BN](/wiki/People/The_Team/Beatmap_Nominators) | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team), melalui situs web [NAT/BN Management](https://bn.mappersguild.com/reports) |
```

File: wiki/Reporting_bad_behaviour/id.md (hunk 3 above)
Comment (Nivalyx) [reply wiki/Reporting_bad_behaviour/id.md:12]: there's a full stop at the end of this sentence in the original article fwiw

image

**Review by Nivalyx — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**